src/components/sections/cta/index.tsx

```diff
@@ -22,6 +22,7 @@ export const CTA = ({
   image,
   variant = 'cta-hero',
   $bgVariant = 'default',
+  $footerClose = false,
```

Comment:
Can we name it `isFooterClose`?
```suggestion
  $isFooterClose = false,
```